deployment/README.md

```diff
@@ -156,9 +156,11 @@ are available:
 
 You run the deployer by instantiating a template. Here is an example with some parameters:
 
-    $ oc process -n openshift logging-deployer-template \
-               -v KIBANA_HOSTNAME=kibana.example.com,PUBLIC_MASTER_URL=https://localhost:8443 \
-               | oc create -f -
+    $ oc new-app logging-deployer-template \
+               -p KIBANA_HOSTNAME=kibana.example.com \
+               -p ES_CLUSTER_SIZE=1 \
+               -p ES_INSTANCE_RAM=1G \
```

Comment:
Sure it does, but this is a readme, so this is just an example. BTW the default is 8G I think...